app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Please you are not using `foreign_key` in the `has_many` associations for `posts` and `comments` in the **User** model. The use of `foreign_key` is important in this context to correctly establish the associations between the models based on the given ERD (Entity-Relationship Diagram) and project requirements.

Reply:
Hello @Owusu-Desmond, thank you for the feedback, the after_save was really helpful, but regarding the last suggestion on the foreign_key. I just added the foreign_key for author_id since is the one that is not following the name convention, the others are so according to the documentation: 
![image](https://github.com/PabloEGonz/blog-app/assets/68977631/47f4ff65-ec19-4045-ac3e-f079ef8f0d92)
It's not necessary to use foreign keys because Rails is already assuming what they are. Furthermore, I already made sure I to set up the FK's in the migrations files.